src/Tasks/Microsoft.NET.Build.Tasks/CheckForDuplicateItems.cs

```diff
@@ -27,11 +27,18 @@ public class CheckForDuplicateItems : TaskBase
         [Required]
         public string MoreInformationLink { get; set; }
 
+        [Output]
+        public ITaskItem [] DeduplicatedItems { get; set; }
+
         protected override void ExecuteCore()
         {
+            DeduplicatedItems = Array.Empty<ITaskItem>();
+
             if (DefaultItemsEnabled && DefaultItemsOfThisTypeEnabled)
             {
-                var duplicateItems = Items.GroupBy(i => i.ItemSpec).Where(g => g.Count() > 1).ToList();
+                var itemGroups = Items.GroupBy(i => i.ItemSpec);
```

Comment:
A customer ran into a similar issue - https://github.com/dotnet/sdk/issues/679. Basically they had different casing for NetStandard.Library and we weren't deduping.